cli/src/commands/branch.rs

```diff
@@ -569,6 +569,59 @@ fn cmd_branch_track(
             names.len()
         )?;
     }
+
+    //show conflicted branches if there are some
+
+    if let Some(mut formatter) = ui.status_formatter() {
+        let template = {
+            let language = workspace_command.commit_template_language()?;
+            let text = command
+                .settings()
+                .config()
+                .get::<String>("templates.branch_list")?;
+            workspace_command
+                .parse_template(&language, &text, CommitTemplateLanguage::wrap_ref_name)?
+                .labeled("branch_list")
+        };
+
+        let mut remote_per_branch: HashMap<String, Vec<String>> = HashMap::new();
+        for n in names.iter() {
+            if !remote_per_branch.contains_key(&n.branch) {
+                remote_per_branch.insert(n.branch.clone(), vec![n.remote.clone()]);
+            } else {
+                remote_per_branch
+                    .get_mut(&n.branch)
+                    .unwrap()
+                    .push(n.remote.clone());
+            }
+        }
+        let branches_to_list =
+            workspace_command
+                .repo()
+                .view()
+                .branches()
+                .filter(|(name, target)| {
+                    remote_per_branch.contains_key(*name) && (target.local_target.has_conflict())
```

Comment:
nit: redundant parens (and `*name` can be just `name`)

Reply:
couldn't replace *name by name as name is &&str